### ISSUE TITLE
VoiceOver Bot Media Seek Slider Tests Failing

### ISSUE THREAD

**Issue by IsaDC** [w3c/aria-at-app#1335]
VoiceOver bot tests for the Media Seek Slider fail with an error after 3/9 tests, canceling the rest.

Logs: https://github.com/bocoup/aria-at-gh-actions-helper/actions/runs/13707999994

**Comment by ccanash** [comment]
Hi @IsaDC  closing this one as it is a duplicate of https://github.com/w3c/aria-at-app/issues/1345
